src/main/scala/akka/contrib/process/BlockingProcess.scala

```diff
@@ -26,7 +25,7 @@ object BlockingProcess {
    * @param stdout a `org.reactivestreams.Publisher` for the standard output stream of the process
    * @param stderr a `org.reactivestreams.Publisher` for the standard error stream of the process
    */
-  case class Started(stdin: Subscriber[ByteString], stdout: Publisher[ByteString], stderr: Publisher[ByteString])
+  case class Started(stdin: Sink[ByteString, Future[Long]], stdout: Source[ByteString, Future[Long]], stderr: Source[ByteString, Future[Long]])
```

Comment:
Should `Sink` require a `Future[Unit]` here, I mean there is no materialized value right...? Same for `Source`.